eda_analysis/eda_analysis.py

```diff
@@ -51,7 +56,7 @@ def describe_na_values(dataframe):
     pass    
 
   
-def describe_cat_var(dataframe,cat_vars):
+def describe_cat_var(dataframe,cat_vars, n_cols = 3):
```

Comment:
I like the idea of using a default `n_cols` argument, but if there is one more input, should the validation check for this input is also needed?

Reply:
This is just setting number of plots to be displayed in one row.This won't have any impact even if extra arguement is provided.

Reply:
I mean if your input is `describe_cat_var(test_data, ['C1'], n_cols=0)` or `describe_cat_var(test_data, ['C1'], n_cols="a")`, you will get an error. So I think checking this input is valid is also needed here.

Reply:
Rightly pointed. Thanks will change add a code for this

Reply:
Fixed it. Have a look